RandomQuestion/MainWindow.xaml

```diff
@@ -7,7 +7,98 @@
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     mc:Ignorable="d">
 
-    <StackPanel Orientation="Horizontal" HorizontalAlignment="Center" VerticalAlignment="Center">
-        <Button x:Name="myButton" Click="myButton_Click">Click Me</Button>
-    </StackPanel>
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="28"/>
+            <RowDefinition Height="*"/>
+        </Grid.RowDefinitions>
+        <StackPanel x:Name="TitleBar"  Grid.Row="0" Margin="0, 0, 120, 0"
+                    Background="{StaticResource backgroundColor}"
+                    Orientation="Horizontal">
+            <Image Source="Assets/rq.gif"/>
+            <TextBlock Margin="10,0,0,0" Foreground="Beige" VerticalAlignment="Center" Text="{x:Bind Title}"/>
+        </StackPanel>
+        <Grid Grid.Row="1">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition/>
+            </Grid.ColumnDefinitions>
+            <Grid.RowDefinitions>
+                <RowDefinition Height="*"/>
+                <RowDefinition Height="60"/>
+            </Grid.RowDefinitions>
+            <Grid.Background>
+                <ImageBrush ImageSource="Assets/qm.png" />
+            </Grid.Background>
+            <Grid Grid.Row="0">
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="*"/>
+                    <ColumnDefinition Width="25*"/>
+                    <ColumnDefinition Width="12*"/>
+                </Grid.ColumnDefinitions>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="4*"/>
+                    <RowDefinition Height="8*"/>
+                    <RowDefinition Height="*"/>
+                </Grid.RowDefinitions>
+                <Grid Grid.Row="1" Grid.Column="1" Name="questionGrid" Visibility="Visible">
```

Comment:
I recommend to keep consistency and move these properties on new lines, like in the other places. It makes the file easier to read.

Reply:
Same for all other places where this applies in the XAML file.

Reply:
Thank you for the suggestion